src/widgets/NumericTextCtrl.cpp

```diff
@@ -1168,7 +1169,7 @@ wxAccStatus NumericTextCtrlAx::GetName(int childId, wxString *name)
             label = mFields[field].label;
          }
 
-         *name = mFields[field - 1].str +
```

Comment:
Pedantically... this commit should be squashed with "Use ParsedNumericFormatConverter in the NumericConverter"

Reply:
I should have noticed when I reviewed that commit.  I am sorry my review was so hasty and careless.